frontend/src/components/Button/Button.styled.ts

```diff
@@ -0,0 +1,75 @@
+import styled, { css } from 'styled-components';
+import { Button as AntButton } from 'antd';
+import { NativeButtonProps } from 'antd/lib/button/button';
+
+interface StyledButtonProps extends NativeButtonProps {
+  variant?: 'primary' | 'secondary';
+  disabled?: boolean;
+  size?: 'small' | 'large';
+  ghost?: boolean;
+}
+
+export const StyledButton = styled(AntButton)<StyledButtonProps>`
+  cursor: pointer;
+  font-size: 1rem;
+  width: 120px;
+  min-height: 30px;
+  border-radius: 5px;
+  border: none;
+  padding: 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  ${(props) =>
+    props.variant === 'primary' &&
+    css`
+      background-color: ${({ theme }) => theme.colors.primary};
+      color: ${({ theme }) => theme.colors.regularText};
+      &:hover {
+        background-color: ${({ theme }) => theme.colors.btnHoverPrimary};
+      }
+      &:focus,
+      &:active {
+        background-color: ${({ theme }) => theme.colors.btnActivePrimary};
+      }
+    `}
```

Comment:
Proponowałbym ustawić to jako domyślne propsy a w innych przypadkach nadpisywać je :)